leapp/utils/report.py

```diff
@@ -1,7 +1,6 @@
 import json
-import six
 
-from jinja2 import Template
```

Comment:
After this we should also remove jinja2 dependencies from spec but not in this PR as it any change has to be synced with leapp-repository.